xprof/btx_timeline.cpp

```diff
@@ -34,8 +34,11 @@ struct timeline_dispatch_s {
   std::unordered_map<h_ddomain_t, perfetto_uuid_t> hp_ddomain2telmtracks;
   std::unordered_map<h_ddomain_t, perfetto_uuid_t> hp_ddomain2cpyalloctracks;
   std::unordered_map<h_dfsdev_t, perfetto_uuid_t> hp_dfsdev2fptracks;
+  std::map<std::pair<std::string, std::string>, perfetto_uuid_t> nicParent2uuid;         // NIC sampling: one parent track per (hostname, interface)
+  std::map<std::tuple<std::string, std::string, std::string>, perfetto_uuid_t> nic2uuid; // NIC sampling: one track per (hostname, interface, counter)
```

Comment:
Can you folop the conversion `hi_` and same for the type `std::string, std::string>` can we do like `hi_t` hostname_interface something like that.